forte/data/data_pack.py

```diff
@@ -866,6 +929,12 @@ def get_data(
                     "currently not supported."
                 )
 
+            if audio_annotation_types:
+                raise NotImplementedError(
+                    "Querying audio annotation types based on ranges is "
+                    "currently not supported."
+                )
```

Comment:
This function fetches data that are covered by an `Annotation` type, that's why Audio annotations cannot be obtained from it, since they are not in the same payload. But it seems like we can extend this function to make it accept `Annotation` or `AudioAnnotation`, and retrieve corresponding data respectively. Could consider making a new issue about this as the next step.